plugin/federation/federation.go

```diff
@@ -4,6 +4,7 @@ import (
 	_ "embed"
 	"errors"
 	"fmt"
+	"github.com/99designs/gqlgen/internal/rewrite"
```

Comment:
Oh thanks!

Reply:
ah must have been auto formatted, let me change that!